collab-document/src/importer/md_importer.rs

```diff
@@ -182,6 +182,11 @@ fn process_mdast_node(
     mdast::Node::Image(image) => process_image(document_data, image, &id),
     _ => {
       trace!("Unhandled node: {:?}", node);
```

Comment:
Should we keep this log as the node has been parsed to the default text paragraph?

Reply:
I think it's ok